tbx/static_src/sass/components/_report-section.scss

```diff
@@ -0,0 +1,78 @@
+@use "config" as *;
+
+// These are used in the body streamfield for the impact report page
+.report-section {
+    margin-bottom: $spacer-mini;
+    margin-top: $spacer-medium;
+
+    &__container {
+        position: relative;
+
+        @include media-query(large) {
+            display: grid;
+            grid-template-columns: repeat(12, 1fr);
+        }
+    }
+
+    &__heading {
+        margin-bottom: $spacer-mini;
+
+        @include media-query(large) {
+            grid-column: 5 / span 6;
+        }
+    }
+
+    &__image {
+        width: 100%;
+    }
+
+    &__picture {
+        display: block;
+        margin-bottom: $spacer-medium;
+        width: 100%;
+    }
+
+    &__rings {
+        @include rings($width: 33px);
+        display: block;
+    }
+
+    &__ring {
+        @include ring($diameter: 30px, $opacity: 0.75);
+
+        &--one {
+            @include ring-one($top: 3px, $color: var(--color--accent-two));
+        }
+
+        &--two {
+            @include ring-two($top: 3px, $color: var(--color--accent-three));
+        }
+    }
+
+    &__short-heading {
+        @include font-size('heading-five');
+        align-items: flex-start;
+        display: flex;
+        font-weight: $weight--semibold;
+        gap: 10px;
+        margin-bottom: $spacer-small;
+
+        @include media-query(large) {
+            margin-bottom: 0;
+            grid-column: 1 / span 3;
+        }
+    }
+
+    &__anchor {
+        position: relative;
+        top: -200px;
```

Comment:
If I'm understanding the purpose of this correctly, perhaps a negative [`scroll-margin`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin-top) might be better?

Reply:
Ha I think you suggested that to me on the current site :) I had a go there and couldn't quite get it working as expected, so I'm going to be very lazy and leave it as is for now as it is working.